public/diagnostic.js

```diff
@@ -23,3 +23,31 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+import Ember from 'ember';
+
+const Order = Ember.Object.extend({
+  name: 'Default',
+  unitPrice: 0.99,
```

Comment:
In order for ember objects to have getters and setters, these properties need to refer to themselves:
```js
name: this.name,
unitPrice: this.unitPrice,
quantity: this.quantity
```